test/uk/gov/hmrc/pdsauthcheckerapi/connectors/PdsConnectorSpec.scala

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.pdsauthcheckerapi.connectors
+
+import com.github.tomakehurst.wiremock.client.WireMock._
+import org.scalatest.wordspec.AnyWordSpec
+import org.scalatest.concurrent.{IntegrationPatience, ScalaFutures}
+import org.scalatest.matchers.should.Matchers
+import play.api.Configuration
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.http.test.{HttpClientV2Support, WireMockSupport}
+import uk.gov.hmrc.pdsauthcheckerapi.base.TestCommonGenerators
+import uk.gov.hmrc.pdsauthcheckerapi.models.{Eori, PdsAuthResponse, PdsAuthResponseResult}
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
+
+import java.time.LocalDate
+import scala.concurrent.ExecutionContext.Implicits.global
+
+
+class PdsConnectorSpec extends AnyWordSpec
```

Comment:
Nicely put together, but can we move integration tests like this into an 'it' directory (a sibling to app and test) as that's typical practice in MDTP